doc/en/doctest.rst

```diff
@@ -64,36 +54,58 @@ and another like this::
         """
         return 42
 
-then you can just invoke ``pytest`` without command line options:
-
-.. code-block:: pytest
+.. code-block:: bash
 
-    $ pytest
+    $ pytest --doctest-modules
     =========================== test session starts ============================
     platform linux -- Python 3.x.y, pytest-4.x.y, py-1.x.y, pluggy-0.x.y
     cachedir: $PYTHON_PREFIX/.pytest_cache
-    rootdir: $REGENDOC_TMPDIR, inifile: pytest.ini
-    collected 1 item
+    rootdir: $REGENDOC_TMPDIR
+    collected 2 items
 
-    mymodule.py .                                                        [100%]
+    mymodule.py .                                                        [ 50%]
+    test_example.txt .                                                   [100%]
 
-    ========================= 1 passed in 0.12 seconds =========================
+    ========================= 2 passed in 0.12 seconds =========================
 
-It is possible to use fixtures using the ``getfixture`` helper:
+You can make these changes permanent in your project by
+putting them into a pytest.ini file like this:
 
-.. code-block:: text
+.. code-block:: ini
 
-    # content of example.rst
-    >>> tmp = getfixture('tmpdir')
-    >>> ...
-    >>>
+    # content of pytest.ini
+    [pytest]
+    addopts = --doctest-modules
 
-Also, :ref:`usefixtures` and :ref:`autouse` fixtures are supported
-when executing text doctest files.
+.. note::
 
-The standard ``doctest`` module provides some setting flags to configure the
-strictness of doctest tests. In pytest, you can enable those flags using the
-configuration file. To make pytest ignore trailing whitespaces and ignore
+    If you are making serious use of doctests through your code, please checkout
+    `Sybil <https://sybil.readthedocs.io/en/latest/index.html>`__, which besides
+    supporting doctests also supports Sphinx directives, plus other features.
+    It provides pytest integration out of the box.
```

Comment:
Might be better to phrase it like this:

```
If you're looking to check that examples in your documentation are correct, including doctests 
`.. codeblock:: python` and any other examples your documentation include, you may wish to
consider `Sybil <https://sybil.readthedocs.io/en/latest/index.html>`__. 
It provides pytest integration out of the box.
```

Reply:
Hmm the first sentence ("If you're looking to check that examples in your documentation are correct") gives the impression that the builtin support does not support this, which reads weird (up until this point we have just saying that it does have some support). Would you mind changing that to "advanced" support or something along those lines? 

This is the documentation for pytest's doctest module, so it makes sense to suggest something which is supported here *plus* a bunch of other features.

```
The builtin pytest doctest supports only `doctest` blocks, but if you are looking 
for more advanced checking over all your documentation examples, 
including doctests, `.. codeblock:: python`, and any other examples your documentation may...
```

What do you think? Please feel free to provide an updated blurb. 😁

Reply:
Yep, your new version is great, go with that :-)

Reply:
Updated. 👍